integration/network/network_test.go

```diff
@@ -89,3 +91,28 @@ func TestNetworkInvalidJSON(t *testing.T) {
 		})
 	}
 }
+
+func TestHostIPv4BridgeLabel(t *testing.T) {
+	skip.If(t, testEnv.OSType == "windows")
+	skip.If(t, testEnv.IsRemoteDaemon)
+	d := daemon.New(t)
+	d.Start(t)
+	defer d.Stop(t)
+	c := d.NewClientT(t)
+	defer c.Close()
+	ctx := context.Background()
+
+	ipv4SNATAddr := "172.0.0.172"
+	// Create a bridge network with --opt com.docker.network.host_ipv4=172.0.0.172
+	bridgeName := "hostIPv4Bridge"
+	network.CreateNoError(context.Background(), t, c, bridgeName,
+		network.WithDriver("bridge"),
+		network.WithOption("com.docker.network.host_ipv4", ipv4SNATAddr),
+		network.WithOption("com.docker.network.bridge.name", bridgeName),
+	)
+	out, err := c.NetworkInspect(ctx, bridgeName, types.NetworkInspectOptions{Verbose: true})
+	assert.NilError(t, err)
+	assert.Assert(t, len(out.IPAM.Config) > 0)
+	// Make sure the SNAT rule exists
+	icmd.RunCommand("iptables", "-t", "nat", "-C", "POSTROUTING", "-s", out.IPAM.Config[0].Subnet, "!", "-o", bridgeName, "-j", "SNAT", "--to-source", ipv4SNATAddr).Assert(t, icmd.Success)
```

Comment:
If you run this in a container with `--net=host` it should be able to (ugh, assuming nft vs iptables isn't an issue) work with remote daemons.... maybe it's not worth it.

Reply:
Its a check command, its not leaving any state behind, and the network should get cleaned once the TC ends

Reply:
Yeah my point was if you use a container (to run the iptables command) you could elide `skip.If(t, testEnv.IsRemoteDaemon)`, but again it may not be worth it just b/c iptables is painful this way.

Reply:
ah I see what you mean now :)